frostsnapp/lib/keygen.dart

```diff
@@ -183,37 +183,32 @@ class _DoKeyGenScreenState extends State<DoKeyGenScreen> {
 
   @override
   Widget build(BuildContext context) {
-    return PopScope(
-      onPopInvoked: (didPop) async {
-        coord.cancelAll();
```

Comment:
We do need to cancel somehow when the back button is pressed though. The problem is that `popScope` also gets triggered when it closes successfully. I had the same problem in #117. I've got an idea I'll try and push a commit.

Reply:
see: 11597f6074b851d7c7e827d9d83cfc2cf0c39a4e